pallets/types.json

```diff
@@ -26,5 +26,70 @@
         "delegator": "AccountId",
         "score": "u64",
         "validators": "Vec<AccountId>"
-    }
+    },
+    "MemberId": "u64",
+  "ProposalId": "u64",
+  "Limits": {
+    "max_tx_value": "u128",
+    "day_max_limit": "u128",
+    "day_max_limit_for_one_address": "u128",
+    "max_pending_tx_limit": "u128",
+    "min_tx_value": "u128"
+  },
+  "Status": {
+    "_enum": [
+      "Revoked",
+      "Pending",
+      "PauseTheBridge",
+      "ResumeTheBridge",
+      "UpdateValidatorSet",
+      "UpdateLimits",
+      "Deposit",
+      "Withdraw",
+      "Approved",
+      "Canceled",
+      "Confirmed"
+    ]
+  },
+  "Kind": {
+    "_enum": [
+      "Transfer",
+      "Limits",
+      "Validator",
+      "Bridge"
+    ]
+  },
+  "TransferMessage": {
+    "message_id": "H256",
+    "eth_address": "H160",
+    "substrate_address": "AccountId",
+    "amount": "TokenBalance",
+    "status": "Status",
+    "direction": "Status"
```

Comment:
I've checked the Rust type in both master and the 2.0 branch and there is no `direction` field in `TransferMessage` struct. The missing field is called `action`

Reply:
I will change it to `action`